js/apps/events-page/services/dateUtilityService.js

```diff
@@ -3,11 +3,11 @@
 
 	const dateUtilityService = () => {
 		const addDays = (dateOrString, daysToAdd) => {
-			const date = typeof dateOrString === 'string' ? new Date(dateOrString) : dateOrString;
+			const date = typeof dateOrString === 'string' ?
+				new Date(dateOrString) :
+				dateOrString;
 
-			if (typeof date !== 'object' || date === 'Invalid Date') {
-				return date;
-			}
+			if (typeof date !== 'object') return date;
```

Comment:
Is lint check OK with you removing curly braces from the if body?  I've had some style checks yell at me for doing this.

Reply:
This is ok per airbnb - https://github.com/airbnb/javascript#blocks--braces